README.md

```diff
@@ -7,15 +7,10 @@ A suitcase subpackage translates between the bluesky document model and a file
 format. This includes:
 
 * exporting from documents to files (potentially lossy)
-* ingesting files into documents (lossless)
-* "reflecting" files into an in-memory representation as documents, on demand
-  (lossless)
-* reading externally-written data to deference pointers in Events --- a.k.a. 
-  databroker's "handlers"
-
-Each suitcase subpackage should do these things for a particular file
-format/layout. Some parameterization is possible, but if two given file layouts
-are substantially different, they should be handled by different subpackages.
+
+Each suitcase subpackage should do this for a particular file format/layout.
```

Comment:
Maybe we can tighten this up even more. How about:

> Each suitcase subpackage should export the bluesky document model to a particular file/format layout.

and then delete, from above:

>  A suitcase subpackage translates between the bluesky document model and a file
> format. This includes:
> * exporting from documents to files (potentially lossy)